doc/classes/TextServer.xml

```diff
@@ -403,7 +403,7 @@
 			<return type="float" />
 			<param index="0" name="font_rid" type="RID" />
 			<description>
-				Deprecated. This method always returns [code]1.0[/code].
+				This method always returns [code]1.0[/code].
```

Comment:
`font_get_global_oversampling` has `does nothing and`, so this one should have it as well.